router.go

```diff
@@ -661,6 +688,8 @@ func Router(addr string) *echo.Echo {
 		g.GET("", getAppsList, jsonEndpoint)
 		g.HEAD("/pending", getPendingVersions, jsonEndpoint)
 		g.GET("/pending", getPendingVersions, jsonEndpoint)
+		g.HEAD("/pending/:app/:version/approval", approvePendingVersion, jsonEndpoint)
+		g.GET("/pending/:app/:version/approval", approvePendingVersion, jsonEndpoint)
```

Comment:
This is a "mutating" endpoint, it should be a `POST` or `PUT`. Maybe a `PUT` is more appropriate here.